File: app/src/main/java/com/demo/firebase/myapplication/RoboTestingActivity.java
package com.demo.firebase.myapplication;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

public class RoboTestingActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.robo_testing);
        Button closeButton = findViewById(R.id.close_button);
        closeButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
    }
}

File: app/src/main/java/com/demo/firebase/myapplication/RoboTestingActivity.kt
package com.demo.firebase.myapplication

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button

class RoboTestingActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.robo_testing)
        val closeButton = findViewById<Button>(R.id.close_button)
        closeButton.setOnClickListener { finish() }
    }
}
